composites/Plugin/SnippetPreview/components/SnippetPreview.js

```diff
@@ -615,6 +566,9 @@ export default class SnippetPreview extends PureComponent {
 		/* eslint-disable jsx-a11y/mouse-events-have-key-events */
 		return (
 			<section>
+				<div>
+					<HelpTextWrapper />
```

Comment:
Please pass the specific help text as a prop here, instead of hardcoding it in the wrapper. This way, the wrapper remains reusable.